scripts/generate_mkdocs.py

```diff
@@ -110,7 +110,8 @@ def generate_rule_metadata(rule_doc: Path) -> None:
     For example:
     ```yaml
     ---
-    description: Checks for abstract classes without abstract methods.
+    description: Checks for abstract classes without abstract methods or abstract class
+    variables.
```

Comment:
Wow, nice grep! Remember to update this again after modifying the docstring.

Reply:
https://github.com/cmp0xff/astral-sh-ruff/commit/680b19969fd234a6cb706bb4e68bdeff103a51b1